src/Illuminate/Support/ServiceProvider.php

```diff
@@ -23,6 +23,16 @@ abstract class ServiceProvider
      */
     protected $defer = false;
 
+    /**
+     * @var array
+     */
+    public $bindings = [];
```

Comment:
Maybe make it `protected`?

Reply:
then we should add the getter

Reply:
Yes, my mistake. Haven't checked that it's used outside of the base provider.